install-deps.sh

```diff
@@ -85,11 +85,19 @@ else
 	if [ -n "$backports" ] ; then rm $control; fi
         ;;
     centos|fedora|rhel)
-        echo "Using yum to install dependencies"
-        $SUDO yum install -y redhat-lsb-core
+        yumdnf="yum"
+        builddepcmd="yum-builddep -y"
+        if test "$(echo "$VERSION_ID >= 22" | bc)" -ne 0; then
```

Comment:
Try it with rhel or CentOS where VERSION_ID may be something like "7.2". Bash is not good at doing float/int comparisons